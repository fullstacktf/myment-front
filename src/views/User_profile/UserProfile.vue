<template>
  <div class="userContainer">
    <div class="profile"></div>
  </div>
</template>

<script>
export default {
  name: UserProfile,
};
</script>

<style>
.userContainer {
  width: 100vw;
  height: 100vh;
  background: rgba(100, 100, 100, 50);
}
</style>
