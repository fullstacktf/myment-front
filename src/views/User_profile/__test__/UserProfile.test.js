import UserProfile from '../UserProfile.vue';
import { shallowMount, createLocalVue } from '@vue/test-utils';
import Buefy from 'buefy';
import mock from '../../../test/mockBack';

describe('UserProfile', () => {
  let localVue;
  beforeEach(() => {
    const Vue = createLocalVue();
    Vue.use(Buefy);
    localVue = Vue;
  });
  test('render component', () => {
    const wrapper = shallowMount(UserProfile, {
      localVue,
    });
    expect(wrapper.html()).toMatchSnapshot();
  });

  test('Check props', () => {});

  test('check Contents props', () => {});
});
