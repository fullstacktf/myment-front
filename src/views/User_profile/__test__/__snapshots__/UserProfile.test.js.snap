// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfile render component 1`] = `"<div class=\\"userContainer\\"><div class=\\"profile\\"></div></div>"`;
